package.json

```diff
@@ -11,20 +11,20 @@
   "author": "Nir Gal",
   "license": "ISC",
   "dependencies": {
-    "@discordjs/opus": "^0.3.3",
-    "@vitalets/google-translate-api": "^5.0.0",
-    "canvas": "^2.6.1",
-    "cheerio": "^1.0.0-rc.5",
+    "@discordjs/opus": "^0.5.0",
+    "@vitalets/google-translate-api": "^7.0.0",
+    "canvas": "^2.8.0",
+    "cheerio": "^1.0.0-rc.9",
     "discord-paginationembed": "^2.1.0",
     "discord.js": "^12.5.3",
     "discord.js-commando": "^0.12.3",
-    "emoji-regex": "^9.2.1",
-    "ffmpeg-static": "^4.2.7",
-    "iso-639-1": "^2.1.8",
-    "probe-image-size": "^6.0.0",
+    "emoji-regex": "^9.2.2",
```

Comment:
wait is this even used in the project ?

Reply:
I can't remember why I added that dependency. It's not in use so it can be removed